### ISSUE TITLE
the emojis do not render properly

### ISSUE THREAD

**Issue by amtoine** [Baptiste-Pollien/pacrust#2]
Hey, it's me again :yum: 

**THE BUG**:
as can be seen in [src/console.rs#L123](https://github.com/Baptiste-Pollien/pacrust/blob/main/src/console.rs#L123), the emojis do not render properly, see [here](https://imgur.com/gallery/iZ54MDc)

maybe there is a special font to install? :open_mouth:

**Comment by Baptiste-Pollien** [comment]
Yes, I think you should use any fonts that support emoji  :wink:

**Comment by amtoine** [comment]
mm i tried with the "mononoki" Nerd Font, which should have a loooot of emojis, but they do not appear :confused:
